roles/ceph-osd/tasks/check_system.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: fail on unsupported {{osd_mkfs_type}} filesystem (configured by osd_mkfs_type variable)
+  command: grep -q "[[:space:]]{{osd_mkfs_type}}$" /proc/filesystems
```

Comment:
Can you do this ASAP? We want to tag stable-2.2 by the end of the week.
Thanks!